plugins/Eq/EqParameterWidget.cpp

```diff
@@ -92,6 +92,7 @@ EqParameterWidget::~EqParameterWidget()
 
 void EqParameterWidget::updateView()
```

Comment:
The `updateView()` name is unfortunate because the method changes the model, but this is not introduced by this request.